tools/xtce_generation/telemetry_generator.py

```diff
@@ -250,9 +250,19 @@ def create_remaining_parameters(self, parameter_set):
 
             description = Et.SubElement(parameter, "xtce:LongDescription")
```

Comment:
@GFMoraga and @tech3371 Do we want to put the short description into the long description tag?

Looking at this and searching through it looks like there is both short and long description supported. https://public.ccsds.org/Pubs/660x2g2.pdf

So, would it make sense to do something like the following so you could have *both* short and/or long descriptions supported?
```suggestion
            if row.get("shortDescription"):
                description = Et.SubElement(parameter, "xtce:ShortDescription")
                description.text = row.get("shortDescription")
            if row.get("longDescription"):
                description = Et.SubElement(parameter, "xtce:LongDescription")
                description.text = row.get("longDescription")
```

Reply:
cc @sdhoyt

Reply:
I think this is a great idea. I know most will have BOTH, but not all will have the descriptions. As long as the this resolves the NaN error in the descriptions that are blank. I will say that the `xtce.xml` file won't look the best with both, and we need to decide if both are required...?